testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/server/nongraceful/deploymenta/ResourceA.java

```diff
@@ -0,0 +1,15 @@
+package org.jboss.as.test.manualmode.server.nongraceful.deploymenta;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+
+@ApplicationScoped
+@Path("/resourcea")
+public class ResourceA {
+    @GET
+    public String get() {
+        System.out.println("***** [non-graceful] Inside ResourceA");
```

Comment:
Please comment out all of these. We want to avoid logging during tests as it puts IO pressure on the CI servers.

Reply:
I forgot all about those. :)